src/OpenTelemetry.Api/.publicApi/netstandard2.0/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,327 @@
+OpenTelemetry.Metrics.Meter.Meter() -> void
+OpenTelemetry.Metrics.MeterFactoryBase
+OpenTelemetry.Metrics.MeterFactoryBase.MeterFactoryBase() -> void
+OpenTelemetry.Metrics.MeterProvider
+OpenTelemetry.Metrics.MeterProvider.Dispose() -> void
+OpenTelemetry.Metrics.MeterProvider.MeterProvider() -> void
+OpenTelemetry.Metrics.NoopBoundCounterMetric<T>
+OpenTelemetry.Metrics.NoopBoundCounterMetric<T>.NoopBoundCounterMetric() -> void
+OpenTelemetry.Metrics.NoopBoundMeasureMetric<T>
+OpenTelemetry.Metrics.NoopBoundMeasureMetric<T>.NoopBoundMeasureMetric() -> void
+OpenTelemetry.Metrics.NoopCounterMetric<T>
+OpenTelemetry.Metrics.NoopCounterMetric<T>.NoopCounterMetric() -> void
+OpenTelemetry.Metrics.NoopDoubleObserverMetric
+OpenTelemetry.Metrics.NoopDoubleObserverMetric.NoopDoubleObserverMetric() -> void
+OpenTelemetry.Metrics.NoopDoubleObserverMetricHandle
+OpenTelemetry.Metrics.NoopDoubleObserverMetricHandle.NoopDoubleObserverMetricHandle() -> void
+OpenTelemetry.Metrics.NoopInt64ObserverMetric
+OpenTelemetry.Metrics.NoopInt64ObserverMetric.NoopInt64ObserverMetric() -> void
+OpenTelemetry.Metrics.NoopInt64ObserverMetricHandle
+OpenTelemetry.Metrics.NoopInt64ObserverMetricHandle.NoopInt64ObserverMetricHandle() -> void
+OpenTelemetry.Metrics.NoopMeasureMetric<T>
+OpenTelemetry.Metrics.NoopMeasureMetric<T>.NoopMeasureMetric() -> void
+OpenTelemetry.Trace.ActivityExtensions
+OpenTelemetry.Trace.IActivityEnumerator<T>
+OpenTelemetry.Trace.IActivityEnumerator<T>.ForEach(T item) -> bool
+OpenTelemetry.Trace.Link
+OpenTelemetry.Trace.Link.Attributes.get -> System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, object>>
+OpenTelemetry.Trace.Link.Context.get -> OpenTelemetry.Trace.SpanContext
+OpenTelemetry.Trace.Link.Equals(OpenTelemetry.Trace.Link other) -> bool
+OpenTelemetry.Trace.Link.Link(in OpenTelemetry.Trace.SpanContext spanContext) -> void
+OpenTelemetry.Trace.Link.Link(in OpenTelemetry.Trace.SpanContext spanContext, OpenTelemetry.Trace.SpanAttributes attributes) -> void
+OpenTelemetry.Trace.SpanAttributeConstants
+OpenTelemetry.Trace.SpanAttributes
+OpenTelemetry.Trace.SpanAttributes.Add(string key, bool value) -> void
+OpenTelemetry.Trace.SpanAttributes.Add(string key, bool[] values) -> void
+OpenTelemetry.Trace.SpanAttributes.Add(string key, double value) -> void
+OpenTelemetry.Trace.SpanAttributes.Add(string key, double[] values) -> void
+OpenTelemetry.Trace.SpanAttributes.Add(string key, long value) -> void
+OpenTelemetry.Trace.SpanAttributes.Add(string key, long[] values) -> void
+OpenTelemetry.Trace.SpanAttributes.Add(string key, string value) -> void
+OpenTelemetry.Trace.SpanAttributes.Add(string key, string[] values) -> void
+OpenTelemetry.Trace.SpanAttributes.SpanAttributes() -> void
+OpenTelemetry.Trace.SpanAttributes.SpanAttributes(System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, object>> attributes) -> void
+OpenTelemetry.Trace.SpanContext
```

Comment:
Is SpanContext still a thing? Do we want ActivityContext instead?

Reply:
`SpanContext` is our struct that uses `ActivityContext`. This was made that way to have the same name as the spec.